spec/models/user_spec.rb

```diff
@@ -51,4 +51,27 @@
       expect(user.signed_in_before).to be false
     end
   end
+
+  describe 'appointments_count' do
+    let (:course1) { create(:course) }
+    let (:course2) { create(:course) }
+    let (:user) { create(:user) }
+
+    it 'returns 0 if the person has no appointments' do
+      expect(user.appointments_count).to eq 0
+    end
+
+    it 'returns 1 if the person has 1 appointment' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.